### ISSUE TITLE
Feature Request: Support for Krowi's Achievement Filter

### ISSUE THREAD

**Issue by Dathwada** [ItsJustMeChris/idTip-Community-Fork#35]
I would like to request the feature that this addon supports the "[Krowi's Achievement Filter](https://www.curseforge.com/wow/addons/krowi-achievement-filter)" addon and the IDs are also displayed in the tooltip when I hover over the achievements.

**Comment by ItsJustMeChris** [comment]
I don't think it necessarily seems complicated, but I would prefer that they utilize idTip's LibStub export if they wish to include this functionality instead of the us having to "reverse engineer" their addon/frame logic to handle them separately from the default achievement UI. I'll leave this open incase anyone else wants to do the work, but personally I'm not quite sure it's worth it in the near future..
